config/main.py

```diff
@@ -33,6 +36,77 @@ def run_command(command, display_cmd=False, ignore_error=False):
     if proc.returncode != 0 and not ignore_error:
         sys.exit(proc.returncode)
 
+
+def interface_alias_to_name(interface_alias):
+    """Return default interface name if alias name is given as argument
+    """
+
+    cmd = 'sonic-cfggen -d --var-json "PORT"'
+    p = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE)
+
+    port_dict = json.loads(p.stdout.read())
+
+    if interface_alias is not None:
+        for port_name in natsorted(port_dict.keys()):
+            if interface_alias == port_dict[port_name]['alias']:
+                return port_name
+        print "Invalid interface {}".format(interface_alias)
+        return None
+
+
+def interface_name_to_alias(interface_name):
+    """Return alias interface name if default name is given as argument
+    """
+
+    cmd = 'sonic-cfggen -d --var-json "PORT"'
+    p = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE)
+
+    port_dict = json.loads(p.stdout.read())
+
+    if interface_name is not None:
+        for port_name in natsorted(port_dict.keys()):
+            if interface_name == port_name:
+                return port_dict[port_name]['alias']
+        print "Invalid interface {}".format(interface_alias)
+        return None
+
+
+def set_interface_mode(mode):
+    """Modify SONIC_CLI_IFACE_MODE env variable in user .bashrc
+    """
+    user = os.getenv('SUDO_USER')
+    bashrc_ifacemode_line = "SONIC_CLI_IFACE_MODE={}".format(mode)
+
+    if not user:
+        user = os.getenv('USER')
+
+    if user != "root":
+        bashrc = "/home/{}/.bashrc".format(user)
+    else:
+        raise click.Abort()
+
+    f = open(bashrc, 'r')
+    filedata = f.read()
+    f.close()
+
+    if "SONIC_CLI_IFACE_MODE" not in filedata:
+        newdata = filedata + bashrc_ifacemode_line
+        newdata += "\n"
```

Comment:
I suggest adding the newline directly to `bashrc_ifacemode_line` on line 78 as follows:

`bashrc_ifacemode_line = "SONIC_CLI_IFACE_MODE={}\n".format(mode)`